source/helpers/FileUpload.ts

```diff
@@ -35,16 +46,14 @@ const uploadFile = async (endpoint, fileName, fileType, fileData, headers = {})
       data: { fileName, mime: `image/${fileType}` },
     });
 
-    // alternative version
-    // const { url, fields } = signedUrlResponse.data.data.attributes;
-
-    const { uploadUrl, uploadedFileName } = signedUrlResponse.data.data.attributes;
+    const { uploadUrl, fileName: uploadedFileName } = signedUrlResponse.data.data.attributes;
```

Comment:
Save to an variabel instead maybe? Since the second parameter name is already is defined and we have to "rename" it.
```
// the name can probably be better but just a suggestion, for the case of it.
const s3Response = ...
s3Response.fileName
```

Reply:
fileUploadAttributes maybe?

Reply:
done